airflow/ui/rules/perfectionist.js

```diff
@@ -109,14 +102,12 @@ export const perfectionistRules =
        *
        * @see [perfectionist/sort-objects](https://perfectionist.dev/rules/sort-objects)
        */
-      [`${perfectionistNamespace}/sort-objects`]: ERROR,
-
-      /**
-       * Enforce sorted attributes in Svelte elements.
-       *
-       * @see [perfectionist/sort-svelte-attributes](https://perfectionist.dev/rules/sort-svelte-attributes)
-       */
-      [`${perfectionistNamespace}/sort-svelte-attributes`]: ERROR,
+      [`${perfectionistNamespace}/sort-objects`]: [
+        ERROR,
+        {
+          type: "natural",
```

Comment:
alphabetical sort was forcing things like [100, 50] instead of the other way around